keymaps/git.cson

```diff
@@ -1,7 +1,8 @@
 '.workspace':
-  'ctrl-9': 'github:toggle-git-tab'
-  'ctrl-8': 'github:toggle-github-tab'
-  'ctrl-(': 'github:toggle-git-tab-focus' # ctrl-shift-9
+  'ctrl-8': 'github:toggle-github-tab-focus'
+  'ctrl-*': 'github:toggle-github-tab' # ctrl-shift-8
```

Comment:
I feel like just making the keybinding `'ctrl-shift-8'` would make it immediately clear what the intention was, and allow the comment to be removed?